### PR TITLE
Add hostname options

### PR DESCRIPTION
**Pull request by hongkheng**
It will be nice to be able to serve the components over the network to help testing in mobile devices.

### CODE REVIEW

**Comment by hongkheng** [comment]
I signed it.

File: src/start_server.js [hunk 1]
```diff
@@ -55,7 +57,7 @@ function startWithPort(options) {
     process.exit(69);
   });
 
-  var baseUrl = util.format('http://localhost:%d/components/%s/', options.port,
+  var baseUrl = util.format('http://'+options.host+':%d/components/%s/', options.port,
```
Comment (ebidel) [inline src/start_server.js]: spacing around +

File: src/start_server.js [hunk 2]
```diff
@@ -30,12 +30,14 @@ function startServer(options) {
...
  * @param {Object} options
  * @param {Number} options.port -- port number
+ * @param {String} options.host -- hostname string
  * @param {String=} options.page -- page path, ex: "/", "/index.html"
  * @param {(String|String[])} options.browser -- names of browser apps to launch
  */
 function startWithPort(options) {
 
   options.port = options.port || 8080;
+  options.host = options.host || "0.0.0.0";
```
Comment (ebidel) [inline src/start_server.js]: let's keep `localhost` as the default

**Comment by justinfagnani** [comment]
Thanks!

**Pull request merged**